src/coreclr/src/gc/gc.h

```diff
@@ -107,7 +107,24 @@ class IGCHeapInternal;
 
 /* misc defines */
 #define LARGE_OBJECT_SIZE ((size_t)(85000))
-#define max_generation 2
+
+enum generation_num
```

Comment:
you could add another one here called uoh_start_generation_num which is (max_generation + 1), that way you don't need (max_generation+1) to indicate the starting of UOH and can just use uoh_start_generation_num...seems a little bit better.

Reply:
I thought about this, but was not sure. Will add `uoh_start_generation`

---
In reply to: [350566918](https://github.com/dotnet/runtime/pull/235#discussion_r350566918) [](ancestors = 350566918)